### DISCUSSION TITLE
[@types/tabulator-tables] Missing types for OptionsRowGrouping

### DISCUSSION THREAD

**Discussion by ortwic** [DefinitelyTyped/DefinitelyTyped#68306] [Issues with a @types package]
See documentation:
https://tabulator.info/docs/5.5/group#main-contents
- `groupStartOpen` can be a `boolen[]`
- `GroupArg` can be an array of functions
- maybe more missing types...

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Fixed in PR #68310
